lerna.json

```diff
@@ -4,7 +4,8 @@
     "./cockpit",
     "./examples/*",
     "./odata-mock",
-    "./openapi-mock"
+    "./openapi-mock",
+    "./app-connector"
   ],
   "version": "0.5.1"
```

Comment:
why the version is 0.5.1 not 0.5.2? Please rebase

Reply:
done